templates/base.html

```diff
@@ -0,0 +1,81 @@
+    <body class="d-flex flex-column h-100 min-vh-100">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="/" >בית</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="/about/">עלינו</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" href="/contact/">יצירת קשר</a>
+                        </li>
+                            {%if not request.user.is_anonymous%}
+                                <span class="navbar-text px-2">
+                                    ברוך הבא, {{request.user}}
+                                </span>
+                            {%endif%}
+                        <li class="nav-item">
+                            {%if request.user.is_anonymous%}
+                                <a class="nav-link" href="/login/">כניסה</a>
+                            {%else%}
+
+                                <a class="nav-link" href="/logout/">יציאה</a>
+                            {%endif%}
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <main class="flex-fill">
+            <div class="container-fluid h-100">
+                <div class="row text-center justify-content-center">
+                    <h1 class="mb-5 mt-5">{% block title %} {%endblock%}</h1>
+                    <div class="col-xxl-10 col-xl-12">
+                        {% block content %}
+                        {% endblock %}
+                    </div>
+                </div>
+            </div>
+        </main>
+        <div class="footer container mb-5 mt-3">
+            <div class="row">
+                <div class="col text-end">
+                    <button class="btn btn-outline-dark bi-chevron-right" onclick="goBack()" >חזרה</button>
+                </div>
+                <div class="col text-start">
+                    {% block bottom %}
+                    {%endblock%}
+                </div>
+            </div>
+        </div>
+        <footer class="footer p-3 bg-light text-center">
+            <div class="container">
+                <span class="text-muted">© 2021
+                        <a class="text-dark" href="https://github.com/dandeduck">dandeduck</a>
+                </span>
+            </div>
+        </footer>
+    <script>
+        function goBack() {
+          window.history.back();

```

Comment:
Although a clever addition, this has a problem. Since you use static templates, going back in history doesn't update the content when needed.

So when creating a new order, and then going back, it will still show the old page (without the new order). In order to see the updated page we need to refresh the page.

This can be acceptable if its a known design, just make sure you are fine with it. If not, make an issue or solve now.

Reply:
Fixed it

Reply:
It's now stuck in a loop. 
The `referrer` is not constant in history. When going back, the referrer changes.

Reply:
I opened an [issue](https://github.com/dandeduck/package-tracking-web/issues/19) to fix this in a later pr

Reply:
An issue was opened #19